includes/definitions/deltaorion.yaml

```diff
@@ -0,0 +1,53 @@
+os: deltaorion
+group: delta
+text: 'Delta Orion Controller'
+type: environment
+icon: delta
+discovery_modules:
+    ntp: false
```

Comment:
Please Remove all these and under poller too. If not needed due to bugs etc

Reply:
Removed.